eng/common/scripts/SetTestPipelineVersion.ps1

```diff
@@ -19,10 +20,13 @@ Foreach ($tags in $latestTags)
   $semVars += $tags.Replace("${PackageName}_", "")
 }
 
-$semVarsSorted = [AzureEngSemanticVersion]::SortVersionStrings($semVars)
-LogDebug "Last Published Version $($semVarsSorted[0])"
+if ($semVars -and $semVars.Length -gt 0)
```

Comment:
Why are we checking $semVars instead of the $latestTags? If the git tag -l comes back empty, $latestTags.Length will be zero, there's no need to process foreach loop that's creating semVars in the first place.